jmclient/jmclient/configure.py

```diff
@@ -244,12 +244,20 @@ def jm_single():
 # spends from unconfirmed inputs, which may then get malleated or double-spent!
 # other counterparties are likely to reject unconfirmed inputs... don't do it.
 
-# options: self, random-peer, not-self.
+# tx_broadcast: options: self, random-peer, not-self.
+#
 # self = broadcast transaction with your own bitcoin node.
+#
 # random-peer = everyone who took part in the coinjoin has a chance of broadcasting
-# not-self = never broadcast with your own ip
 # note: if your counterparties do not support it, you will fall back
 # to broadcasting via your own node.
+#
+# not-self = never broadcast with your own ip
```

Comment:
Should this become `never broadcast with your own bitcoin node` to be consistent with the `self` option?
Also arguably more correct (?) since e.g., Tor users don't exactly use their own ip but still could benefit from this option.

Reply:
Yeah, sure, fair point.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/004945f038ab68d7b2d87ad9fe073b3b80844c78